src/org/pahappa/systems/ticketing/models/Ticket.java

```diff
@@ -1,11 +1,30 @@
 package org.pahappa.systems.ticketing.models;
 
+import org.pahappa.systems.ticketing.constants.TicketCategory;
+import org.pahappa.systems.ticketing.constants.TicketStatus;
+
 /**
  * A ticket refers to a unit of work or a request that is submitted
  * by a user or customer to seek assistance, report an issue, or request a service.
  * It serves as a record or a container that captures all the relevant information
  * related to the user's request or issue.
  */
 public class Ticket {
+    public TicketStatus status;
+    public String tktNumber = null;
+    public String customerID;
+    public String contact;
+    public String issueDesc;
```

Comment:
Kindly follow this guide on how to implement encapsulation in this class.

https://beginnersbook.com/2013/05/encapsulation-in-java/